File: app/src/main/java/com/artamonov/millionplanets/fight/FightActivityView.java
package com.artamonov.millionplanets.fight;

import com.artamonov.millionplanets.model.User;

public interface FightActivityView {

    void calculateLoot(String nicknameGet, String nicknameLose, String ship);

    void setFightLog(int hpDamage, int shieldDamage, int enemyHpDamage, int enemyShieldDamage);

    void setProgressBar(boolean state);

    void setUserData(User userList);

    void setEnemyData(User userList);

    void showLootSnackbar(Boolean isYouWon, String ship);

    void showYouWonMessage();

    void showEnemyWonMessage();

    void startTimer();
}

File: app/src/main/java/com/artamonov/millionplanets/fight/FightActivityView.kt
package com.artamonov.millionplanets.fight

import com.artamonov.millionplanets.model.User

interface FightActivityView {
    fun calculateLoot(nicknameGet: String, nicknameLose: String, ship: String)
    fun setFightLog(hpDamage: Int, shieldDamage: Int, enemyHpDamage: Int, enemyShieldDamage: Int)
    fun setProgressBar(state: Boolean)
    fun setUserData(userList: User)
    fun setEnemyData(userList: User)
    fun showLootSnackbar(isYouWon: Boolean, ship: String)
    fun showYouWonMessage()
    fun showEnemyWonMessage()
    fun startTimer()
}